final_prep/README.md

```diff
@@ -44,31 +44,32 @@ In Mod 0 you've learned about different techniques for managing your time at Tur
 
 When you are finished, add screenshots of your calendar so we can provide feedback if needed!
 
-- `Add Week 1 Screenshot Here`
-- `Add Week 2 Screenshot Here`
-- `Add Week 3 Screenshot Here`
+- `Add Week 1 Screenshot Here` <img width="1204" alt="Screen Shot 2022-02-12 at 1 17 58 PM" src="https://user-images.githubusercontent.com/98674727/153728804-0763b70e-9c59-462f-8e9f-701d4a08d075.png">
+- `Add Week 2 Screenshot Here`<img width="1209" alt="Screen Shot 2022-02-12 at 1 19 23 PM" src="https://user-images.githubusercontent.com/98674727/153728814-adc440d2-20e6-4c52-94b1-c29ab14118a7.png">
+- `Add Week 3 Screenshot Here` <img width="1211" alt="Screen Shot 2022-02-12 at 1 19 36 PM" src="https://user-images.githubusercontent.com/98674727/153728829-51220745-119c-493a-a454-9283b150afc0.png">
 
 ## Mentorship Prep
 
 Mentorship is an integral part of the Turing experience and will help jumpstart your technical career. In order to get your mentor relationship started on the right foot, please complete the following deliverables:
 
 - [ ] [Create a profile on Turing Community Connect by following these directions](https://docs.google.com/document/d/1vpyKGu92l1HGkJzULNcyyE72946f4QO1DhQgIz3v1E0/edit?usp=sharing)
 - [ ] Complete the [Mentorship DTR Prep](https://gist.github.com/ericweissman/51965bdcbf42970d43d817818bfaef3c)
-  - [ ] Add link to your gist here: 
+  - [ ] Add link to your gist here: [Casey's Mentee DTR](https://gist.github.com/casefaz/5f5ea03e742987f53240da413b2558c0)
```

Comment:
Nice work on your mentorship prep! Definitely refer back to this gist when you get introduced / meet your mentor in a few weeks.